components/InfoTooltip.tsx

```diff
@@ -0,0 +1,37 @@
+import { InfoOutlineIcon } from '@chakra-ui/icons'
```

Comment:
More for above file 
do you think we should remove the husky stuff for now ? 
I added it to help ensure people are linting and doing some code quality like ensuring the project builds for netlify but we have gotten very good at this already. 

Any thoughts @worrex

Reply:
i think its good to have, i just have problems with pushing with github desktop and deleting fixes this. sometimes i forget to exclude it from the commit.

Reply:
I think it's good to have that qa layer

Reply:
@0xFable I think the flow creator can close their incentive flow anytime, can't they ?

Reply:
@nick134-bit  can you do proper capitalization for the tooltips